tests/utils/dockercli/VolumeCreationTestUtil.go

```diff
@@ -26,26 +26,48 @@ import (
 
 var SSH_OPTS []string = []string{strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[0], strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[1], "-q", "-kTax", "-o StrictHostKeyChecking=no"}
 
-// This util method is going to create vsphere docker volume with
+// CreateDefaultVolume - create vsphere docker volume with
 // defaults.
 func CreateDefaultVolume(ip string, name string) ([]byte, error) {
-	fmt.Printf("\ncreating volume [%s] on VM[%s]", name, ip)
+	fmt.Printf("Creating volume [%s] on VM [%s]\n", name, ip)
 	return InvokeCommand(ip, "docker volume create --driver=vsphere --name="+name)
 }
 
-// This helper deletes the created volume as per passed volume name.
+// AttachVolume - attach volume to container on given host
+func AttachVolume(host, volName, containerName string) ([]byte, error) {
+	fmt.Printf("Attaching volume [%s] on VM[%s]\n", volName, host)
+	return InvokeCommand(host, "docker run -d -v "+volName+
+		":/vol1 --name "+containerName+
+		" busybox tail -f /dev/null")
+}
+
+// DeleteVolume - deletes the created volume as per passed volume name.
 func DeleteVolume(name string, ip string) ([]byte, error) {
```

Comment:
I noticed that the APIs defined in these utils are not consistent: some APIs have hostName (or ip) as the 1st parameter, some others have it as the 2nd or last parameter. This inconsistency makes the APIs difficult to use. Can we fix all of them together?